### ISSUE TITLE
Assigned drop down is blank

### ISSUE THREAD

**Issue by robinlennox** [dunbarcyber/cyclops#20]
When I try select an assignee the dropdown is blank.
Screenshot of blank dropdown
[
![assigned_blank](https://user-images.githubusercontent.com/5493700/30139051-cd1eb202-93bf-11e7-819c-90fa7c26c46d.jpg)
](url)

**Comment by chasepbrewer** [comment]
Have you given your User's a first and/or last name in the admin panel? This typically happens to me when I have to reset my developer database and rename my user's again.

**Comment by robinlennox** [comment]
Thanks for the feedback, it would be good it there is no first name to default to the email address.

**Comment by chasepbrewer** [comment]
Yeah that's a good idea. I thought the same thing after you submitted the issue. I just got used to it for so long that it didn't even bother me for a while. I'm adding an enhancement label to this issue for me to fix that.
